src/mon/MDSMonitor.cc

```diff
@@ -1973,12 +1988,13 @@ bool MDSMonitor::maybe_expand_cluster(std::shared_ptr<Filesystem> fs)
     }
 
     const auto &new_info = pending_fsmap.get_info_gid(newgid);
-    dout(1) << "adding standby " << new_info.addr
+    dout(1) << "assigned standby " << new_info.addr
             << " as mds." << mds << dendl;
 
-    mon->clog->info() << "Assigned MDS " << new_info.name << " to filesystem "
-                      << fs->mds_map.fs_name << " as rank " << mds << " (now "
-                      << "has " << fs->mds_map.get_num_in_mds() << " ranks)";
+    mon->clog->info() << "Assigned " << new_info.human_name() << " to "
```

Comment:
nitpick: other clog messages begin with the mds human name. Instead do `new_info.human_name() <<  "assigned to..."`